salt/cache/mysql_cache.py

```diff
@@ -99,7 +97,14 @@ def run_query(conn, query, retries=3):
     """
     try:
         cur = conn.cursor()
-        out = cur.execute(query)
+
+        if args is None or args == {}:
+            log.debug("Doing query: %s", query)
+            out = cur.execute(query)
+        else:
+            log.debug("Doing query: %s args: %s ", query, repr(args))
```

Comment:
here too

Reply:
FWIW - when we toggle debug logging we warn at the top of the log that sensitive information (like passwords) may be logged

Reply:
I still don't think we should log them on purpose.

Reply:
@Ch3LL  I share the concern about this showing sensitive information but this is something that could be useful when debugging problems with the MySQL cache module, and since it's using the `debug` level most users won't see this.

Reply:
I still think we should prioritize ensuring we do not log sensitive information. Where would the sensitive information show up? In the args? Could we parse out the args to not include the password and username?

Reply:
Dug a bit deeper and the `args` in the `run_query` function are only passed via the `store` function.  They are the cache bank, key, and data values.  The connection data is provided in the `init_client` function, which stores the options in the `__context__` dictionary for later use.  There is a log message when it connects, but the password is set as hidden.  Because of this, I don't believe we need to worry about sanitizing the data that is being logged int he `run_query` function.